### DISCUSSION TITLE
Import csv to database

### DISCUSSION THREAD

**Discussion by pixeluvw** [appwrite/appwrite#3259] [Q&A] [answered]
Hi everyone,

Is it possible to import a csv file to console database? Any idea how to do it?

**Comment by Meldiron** [comment, marked as answer]
You could write a script using one of our [Server SDKs](https://appwrite.io/docs/getting-started-for-server). Alternatively, you can use some community-made tools to do it, for instance, [Tanner's Data Migrator](https://github.com/tannermeade/data-migrator).

**Reply by tannermeade** [reply]
@pixeluvw I just updated DataMigrator to work with the most recent Appwrite version, added Windows support, and just pushed a commit that adds the ability to upload users.

**Comment by leamandeep** [comment]
it's such a pain. You need to use server SDK for uploading database files in Appwrite. Why there isn't any uploader on the appwrite cloud?

**Reply by rafagazani** [reply]
Maybe it's helpful: https://github.com/rafagazani/csv_to_documents.
An example that you can do using the SDK + Cloud Function AppWrite
